src/components/IsImportant/IsImportant.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import "./IsImportant.css"
+
+function IsImportant({ ...props }) {
+
+    return (
+        <span className='important'>!</span>
+    );
+
+}
+
+export default IsImportant;
```

Comment:
It's too early for another component yet.
It does nothing except increasing code complexity.

Reply:
You also shouldn't name your components this way.
`isOpened`, `isExpanded`, ... are good for properties and usually shows how component should looks like.

Thinking about components you should imagine instance of something. E.x. button, checkbox, header, tile, list, photo, icon, etc

Reply:
Let's move it back to Task component according to KISS principle.
https://www.freecodecamp.org/news/keep-it-simple-stupid-how-to-use-the-kiss-principle-in-design/